Lib/json/tool.py

```diff
@@ -25,24 +25,35 @@ def main():
                         help='a JSON file to be validated or pretty-printed')
     parser.add_argument('outfile', nargs='?', type=argparse.FileType('w'),
                         help='write the output of infile to outfile')
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--indent', default=4, type=int,
+                       help='Indent level (number of spaces) for '
+                       'pretty-printing. Defaults to 4.')
+    group.add_argument('--no-indent', action='store_const', dest='indent',
```

Comment:
> Spaces after comma and semicolon are not just for readability. They make JSON compatible with YAML 1.0.

@serhiy-storchaka great point. I think this is a strong argument for not replacing `--no-indent` with `--compact`. However, should we add `--compact` as a third option? I can see the utility for users who wan't the smallest file sizes.